libs/styles/breakpoint-theme.css

```
@@ -0,0 +1,193 @@
+
+  
+/* Static Links */
+.con-block.static-links[class*="dark-"] a:not(.con-button),
+.static-links .con-block[class*="dark-"] a:not(.con-button),
+.con-block.static-links[class*="light-"] a:not(.con-button),
+.static-links .con-block[class*="light-"] a:not(.con-button),
+.con-block.static-links[class*="dark-"] a:not(.con-button):hover,
+.static-links .con-block[class*="dark-"] a:not(.con-button):hover,
+.con-block.static-links[class*="light-"] a:not(.con-button):hover,
+.static-links .con-block[class*="light-"] a:not(.con-button):hover {
+  color: inherit;
+}
+
+/* mobile ONLY */
+@media (max-width: 600px) {
+  /* mobile-only theme */
+  .con-block.light-mobile-only { color: var(--text-color); }
+  .con-block.dark-mobile-only {  color: var(--color-white); }
+
+  .con-block.light-mobile-only a:not(.con-button) { color: var(--link-color); }
+  .con-block.dark-mobile-only a:not(.con-button) { color: var(--link-color-dark); }
+
+  .con-block.light-mobile-only a:not(.con-button):hover,
+  .con-block.light-mobile-only a:not(.con-button):focus,
+  .con-block.light-mobile-only a:not(.con-button):active { color: var(--link-hover-color); }
```

Could we trim these down with `:is`?
```suggestion
  .con-block.light-mobile-only a:not(.con-button):is(:hover, :focus, :active) { color: var(--link-hover-color); }
```

---

Certainly, I've gone thru this css and reduced some bloat using `:is` Tested and checks out.